t/venom/test_suites/test_venom/20_testVenom.yml

```diff
@@ -0,0 +1,15 @@
+name: testsuite with a user executor in custom dir which has multiple steps
+testcases:
+- name: testfoobar multisteps custom
+  steps:
+  - type: foobarcustommultisteps
+    assertions:
+    - result.foobar ShouldEqual "hello world"
+    - result.contento ShouldEqual "hello"
+
+- name: testfoobar multisteps custom
+  steps:
+  - type: foobarcustommultisteps
+    assertions:
+    - result.foobar ShouldEqual "hello world"
+    - content ShouldEqual "hello"
```

Comment:
This assertion failed on my side because `content` is not visible outside executor, we should update assertion.